public/views/InterviewForm.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MindSet</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/interviews.css">
+    <link rel="stylesheet" href="../styles/modal.css">
+    <script async src="../js/index.js"></script>
+    <script async src="../js/interviewsForm.js"></script>
+    <script async src="../js/modal.js"></script>
+  </head>
+  <body>
+    <header>
+      <a id="mindSetLogo" href="../index.html"><span>Mind</span>SET</a>
+      <nav>
+        <a id="adminsNav" href="/views/adminList.html">admins</a>
+        <a id="applicationsNav" href="/views/applicationList.html">applications</a>
+        <a id="clientsNav" href="/views/clientList.html">clients</a>
+        <a id="interviewsNav" href="/views/interviewList.html">interviews</a>
+        <a id="positionsNav" href="/views/positionList.html">positions</a>
+        <a id="profilesNav" href="/views/profileList.html">profiles</a>
+        <a id="psychologistsNav" href="/views/psychologistList.html">psychologists</a>
+        <a id="sessionsNav" href="/views/sessionList.html">sessions</a>
+      </nav>
+    </header>
+
+    <div class="container">
+      <h2 class="header">Interview Form</h2>
+      <form id="form">
+        <label for="postulantName"><span>Postulant Name</span></label>
+        <select name="postulantName" type="text" id="postulantName" required></select>
+        
+        <label for="clientName"><span>Client Name</span></label>
+        <select name="clientName" type="text" id="clientName" required></select>
+        
+        <label for="applicationId"><span>application ID</span></label>
+        <select name="applicationId" type="text" id="applicationId" required></select>
+        
+        <label for="status"><span>status</span></label>
+        <select name="status" type="text" id="status" required></select>
+        
+        <label for="date"><span>Date</span></label>
```

Comment:
use `<input id="date" name="date" type="datetime-local" required/>`

See how it was used [here](https://github.com/BaSD-2021/Esteban-MindSet-server/blob/development/public/views/sessionForm.html#L30)